src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -106,7 +106,11 @@ void MulticopterLandDetector::_update_topics()
 		hover_thrust_estimate_s hte;
 
 		if (_hover_thrust_estimate_sub.update(&hte)) {
-			_params.hoverThrottle = hte.hover_thrust;
+			if (hte.valid) {
```

Comment:
If the user switches from alt/pos mode to attitude, HTE will stop publishing, but `_hover_thrust_estimate_valid` might stay valid. I don't think it's a bit issue, but maybe we need to do something about it...

Reply:
Yes and it's actually a bit of an intentional hole that's assuming continuing to use the once valid hover thrust is going to be less wrong than reverting. Let me know if you have something else in mind.